README.md

```diff
@@ -30,5 +30,5 @@ cargo install mdbook mdbook-katex mdbook-toc mdbook-linkcheck mdbook-mermaid
 ### Serving the book locally
 
 ```sh
```

Comment:
Can we remove the direction to install `mdbook-toc` below? No longer needed.

Reply:
Also would be great to add a section reminding folks to run `just lint-specs-toc-check` before submitting PRs. I removed this section from `linting.md` too, will need to be added back.